bionic/code_references.py

```diff
@@ -151,6 +151,17 @@ def set_tos(t):
 
     for op in dis.get_instructions(code):
         try:
+            if op.opname not in SUPPORTED_INSTRUCTIONS:
+                message = f"""
+                Bytecode operation {op.opname} not found in the list of Bionic
```

Comment:
I don't think the "list of supported bytecode instructions" is a concept that users should have to think about. Either (a) they are using a newer version of Python that we don't fully support, or (b) we screwed up and missed an instruction. Can we look at the Python version, figure out which case it is, and then give the appropriate message?